current_sensor.h

```diff
@@ -0,0 +1,96 @@
+#ifndef __CURRENT_SENSOR__
+#define __CURRENT_SENSOR__
+
+struct PowerData{
+    double current;
+    double voltage;
+    double watts;
+};
+
+/* Init power data structure pointer */
+static PowerData *pwrDataPtr;
+/* Set INA260 Address */
+static INA260 ina260(INA260::ADDR_GND,INA260::ADDR_GND);
+
+static bool setConfig()
```

Comment:
will this function be imported with when importing this file?
setConfig is to general name
set inaConfig
or configIna